.bazelignore

```diff
@@ -1,4 +1,5 @@
 # These are fetched as external repositories.
 third_party/abseil-cpp
 third_party/googletest
+third_party/utf8_range
```

Comment:
We recently moved utf8_range into this repo, so third_party/utf8_range needs to be visible to Bazel.

Reply:
done. PTAL @acozzette